examples/mesh-gateways/datacenter/variables.tf

```diff
@@ -0,0 +1,90 @@
+variable "name" {
+  description = "Name to be used on all the resources as identifier."
+  type        = string
+}
+
+variable "datacenter" {
```

Comment:
What do you think about making this `consul_datacenter` [to match mesh task](https://github.com/hashicorp/terraform-aws-consul-ecs/blob/main/modules/mesh-task/variables.tf#L298)?